include/swift/AST/DiagnosticsSema.def

```diff
@@ -1908,6 +1908,10 @@ ERROR(inheritance_from_final_class,none,
 ERROR(inheritance_from_unspecialized_objc_generic_class,none,
       "inheritance from a generic Objective-C class %0 must bind "
       "type parameters of %0 to specific concrete types", (Identifier))
+ERROR(inheritance_from_class_with_missing_vtable_entries,none,
+      "cannot inherit from class %0 in Swift %1 mode because it has "
+      "overridable members that cannot be loaded",
```

Comment:
Can the diagnostic be phrased in a more user-comprehensible way? "Cannot be loaded" isn't going to mean much to most people, and sounds like a condition that could be the user's fault. Can we describe the reason why the member couldn't be loaded (SDK change between language versions, etc.)?